lang/src/org/partiql/lang/eval/visitors/GroupKeyReferencesVisitorTransform.kt

```diff
@@ -47,6 +47,10 @@ import org.partiql.lang.planner.transforms.errAstNotNormalized
  *
  * ```
  */
+@Deprecated(
+    "This class is subject to removal.",
+    level = DeprecationLevel.WARNING
+)
```

Comment:
Have we released this yet? Is it ok to just remove the class?

Reply:
Released in 0.8.0